nemo/collections/common/data/lhotse/nemo_adapters.py

```diff
@@ -93,6 +102,29 @@ def __len__(self) -> int:
     def __add__(self, other):
         return LazyIteratorChain(self, other)
 
+    def _create_recording(self, audio_path: str, duration: float, sampling_rate: int | None = None,) -> Recording:
+        if sampling_rate is not None:
+            # TODO(pzelasko): It will only work with single-channel audio in the current shape.
```

Comment:
How about integrating channel_selector here ?

Reply:
Is this option documented somewhere? I was looking for something about multi channel but couldn't find anything.

Reply:
Hmm it's there as a utility function in most transcribe() functions

Reply:
I see, NeMo manifests don't have a key with information about the number of channels. That's a bummer. I'll take a closer look at multi-channel Nemo+Lhotse later.